src/main/resources/application.yml

```
@@ -19,3 +19,10 @@ stream:
     member-joined: ${MEMBER_JOINED_GROUP}
   consumer:
     member-joined: ${MEMBER_JOINED_GROUP}-${CONTAINER_NAME}
+
+security:
+  cors:
+    paths:
+      - ${APP_FRONT}
+      - ${CREATOR_FRONT}
+      - ${ADMIN_FRONT}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)